pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -47,6 +47,8 @@ const (
 	ControllerName               = "istio.io/gateway-controller"
 	gatewayAliasForAnnotationKey = "gateway.istio.io/alias-for"
 	gatewayTLSTerminateModeKey   = "gateway.istio.io/tls-terminate-mode"
+	gatewayNameOverride          = "gateway.istio.io/name-override"
+	gatewaySAOverrice            = "istio.io/service-account"
```

Comment:
```suggestion
	gatewaySAOverrice            = "gateway.istio.io/service-account"
```

for consistency

Reply:
And probably `Overrice` -> `Override`

Reply:
> And probably `Overrice` -> `Override`

I kind of like it :-)

Reply:
@howardjohn I'm fine with the suggestion, I copied the annotation name from [waypoint_controller.go](https://github.com/istio/istio/blob/experimental-ambient/pilot/pkg/ambient/controller/waypoint_controller.go#L155) - assuming when I cherrypick this over waypoint should use the same `gateway.istio.io` annotation?